bin/git-navigate-branch

```diff
@@ -26,7 +26,8 @@ done
 
 BRANCH=${1:-$(git rev-parse --abbrev-ref HEAD)}
 
-git config --local alias.prev "checkout HEAD^1"
-git config --local alias.next "!sh -c 'git log --reverse --pretty=%H ${BRANCH} | awk \"/\$(git rev-parse HEAD)/{getline;print}\" | xargs git checkout'"
+git config --local alias.prev "!f() { git checkout HEAD~${1:-1} ;}; f"
+git config --local alias.next "!f() { git rev-list --reverse HEAD..${BRANCH} | sed \"\${1:-1}q;d\" | xargs git checkout ;}; f"
 git config --local alias.last "checkout ${BRANCH}"
-git config --local alias.first "!sh -c 'git checkout \$(git rev-list --reverse ${BRANCH} | head -1)'"
+git config --local alias.first "!f() { git checkout \$(git rev-list --reverse ${BRANCH} | head -1) ;}; f"
+git config --local alias.origin "checkout origin/${BRANCH}"
```

Comment:
Hello Maël. Thanks for the contribution !
I'm kind of conflicted about the "origin" alias. I think the name can create some confusion ("origin" as a very specific meaning in git) and I'm not sure of the use case. In what context would you use it ?

Reply:
Yes, you were right.
It was late, I thought it would be useful to add that "origin" jump in history but it doesn't add much and it's too tied to the "origin" alias name.

I fixed-up the commit by removing this alias.